src/core/messenger/background.ts

```diff
@@ -184,6 +184,8 @@ const setupBackground = async (
 		specs.scrollType,
 	);
 
+	const nativeTemplateId = 11885667;
```

Comment:
```suggestion
	const interscrollerTemplateId = 11885667;
```
Doesn't hurt to be specific here!